### ISSUE TITLE
500 Internal Server Error

### ISSUE THREAD

**Issue by robbrandt** [zikula-modules/Content#301]
New installation of Content-master on Zikula 2.0.11, PHP 5.6.37-1+ubuntu16.04.1+deb.sury.org+1

Immediately after installation I go to /content/admin/pages and get
Oops! An Error Occurred
The server returned a "500 Internal Server Error".
Something is broken. Please let us know what you were doing when this error occurred. We will fix it as soon as possible. Sorry for any inconvenience caused.

Apache error log says:

[Wed Sep 26 04:12:05.566852 2018] [:error] [pid 25805] [client 68.6.112.199:63778] PHP Fatal error:  require_once(): Failed opening required '/home/sitename/public_html/modules/zikula/ContentModule/Base/../vendor/autoload.php' (include_path='.:/usr/share/php') in /home/sitename/public_html/modules/zikula/ContentModule/Base/bootstrap.php on line 19, referer: http://sitename.org/extensions/module/list?justinstalled=%5B20%5D

**Comment by robbrandt** [comment]
I get the same error on MU/News, so this might be a systemic problem that manifests in these modules.

**Comment by Guite** [comment]
It looks like you just downloaded the modules from GitHub. You need to execute `composer install --no-dev` then because vendors are not stored inside the repositories.

**Comment by rallek** [comment]
And you should store the modules inside the correct folder. See install instructions.

**Comment by robbrandt** [comment]
Thanks, problem solved.

I shouldn't have been surprised to need to run composer for Content since I downloaded master, not a release.  But for News I am certain I downloaded the 1.2.1 release.  So must we run composer for ANY module to be installed?

**Comment by Guite** [comment]
> So must we run composer for ANY module to be installed?

No, just for those which require vendors and do not ship them. For News specifically you would need to ask @portugao whether he included vendors in the release.

**Comment by Portugao** [comment]
News 1.2.1 should contain the vendor.

**Comment by robbrandt** [comment]
@Portugao no sir, no vendor dir in the News-1.2.1.tar.gz release.

**Comment by Portugao** [comment]
Hmmhmm...that is not, what I want.
Try to fix it....

**Comment by Portugao** [comment]
Fixed
